requirements.txt

```diff
@@ -29,6 +30,7 @@ schema==0.7.5
 semver==2.13.0
 six==1.16.0
 sql-metadata==2.7.0
+sqlparse==0.4.3 ; python_version >= '3.5'
```

Comment:
whoops, this one's my indirect dependency, I assume? How should I have known to add this?